include/exotic/cester.h

```diff
@@ -3181,6 +3181,7 @@ static __CESTER_INLINE__ void cester_run_test(TestInstance *test_instance, TestC
 #else
         #pragma message("Isolated tests not supported in this environment. The tests will be run on the main process")
         last_status = cester_run_test_no_isolation(test_instance, a_test_case, index);
+#define CASTER_NO_SUBPROCESS 1

```

Comment:
The macro should start with CESTER_, not CASTER_

Reply:
Done.